Marlin/src/gcode/geometry/G92.cpp

```diff
@@ -93,10 +93,37 @@ void GcodeSuite::G92() {
       COPY(coordinate_system[active_coordinate_system], position_shift);
   #endif
 
-  bool didA = parser.seenval('A');
-  if (didA) {
-    const float curr_z = current_position[Z_AXIS];
-    const float adjust_z = parser.value_axis_units(Z_AXIS);
+  bool doZAdjust = false;
+  float adjust_z = 0.0;
+
+
+
+  if(parser.seenval('A')){
+    adjust_z = parser.value_axis_units(Z_AXIS);
+    doZAdjust = true;
+  }else if(parser.seenval('B')){
+    float final_z = parser.value_axis_units(Z_AXIS);
+
+
+    SERIAL_ECHO_START();
```

Comment:
If these echo statements are for debugging and development purposes, please remove them.

Reply:
these are not just for debugging, it provides user feedback